content/news/XXXX-XX-XX-mixxx-2-3-1-released.md

```diff
@@ -0,0 +1,19 @@
+title: Mixxx 2.3.1 released
+authors: Daniel Schürmann
+tags: 2.3, 2.3.1, release announcement
+status: draft
+comments: yes
+
+We are happy to announce first bugfix release of [Mixxx 2.3.1](https://github.com/mixxxdj/mixxx/releases/tag/2.3.1)!
+Head over to the [download page]({filename}/pages/download.md#stable) and download it on Windows, macOS and Linux.
+
+This release comes with new and updated controller mappings, fixes packaging issues and includes some bugfixes.
+
+Mixxx now includes mappings for the [Numark DJ2GO2 Touch](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_dj2go2_touch.html) controller and the [Numark Mixtrack Pro FX](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_mixtrack_pro_fx.html) controller. The [Behringer DDM4000](https://manual.mixxx.org/2.3/en/hardware/controllers/behringer_ddm4000.html), the [Hercules Inpulse 300](https://manual.mixxx.org/2.3/en/hardware/controllers/hercules_djcontrol_inpulse_300.html) mapping, the [Denon MC7000](https://manual.mixxx.org/2.3/en/hardware/controllers/denon_mc7000.html) mapping and the [Traktor S3](https://manual.mixxx.org/2.3/en/hardware/controllers/native_instruments_traktor_kontrol_s3.html) mapping have been improved with Mixxx 2.3.
```

Comment:
```suggestion
Mixxx now includes mappings for the [Numark DJ2GO2 Touch](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_dj2go2_touch.html) controller and the [Numark Mixtrack Pro FX](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_mixtrack_pro_fx.html) controller. The [Behringer DDM4000](https://manual.mixxx.org/2.3/en/hardware/controllers/behringer_ddm4000.html), the [Hercules Inpulse 300](https://manual.mixxx.org/2.3/en/hardware/controllers/hercules_djcontrol_inpulse_300.html) mapping, the [Denon MC7000](https://manual.mixxx.org/2.3/en/hardware/controllers/denon_mc7000.html) mapping and the [Traktor Kontrol S3](https://manual.mixxx.org/2.3/en/hardware/controllers/native_instruments_traktor_kontrol_s3.html) mapping have been improved since Mixxx 2.3.0.
```